third_party/move/move-vm/runtime/src/tracing.rs

```diff
@@ -80,9 +76,7 @@ pub(crate) fn trace(
                 pc,
             ))
             .unwrap();
-        if *SINGLE_STEP_FLUSHING {
-            buf_writer.flush().unwrap();
-        }
+        buf_writer.flush().unwrap();
```

Comment:
Always flush to file. We don't have the option to flush the file towards the end of execution.